### DISCUSSION TITLE
Step-by-step demo for endoscopy sample app

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/MONAILabel#1033] [Q&A]
Hi! Can we have a step-by-step demo recording for using endoscopy sample app (including setting up CVAT) like the one for using DSA https://drive.google.com/file/d/19j7CMx7c8nn96ZL75L1a26usw8pHW-Sj/view?usp=sharing? That DSA recording is super helpful! Thanks!

**Comment by diazandr3s** [comment]
Hi @hw-ju,

Thanks for this suggestion.
We're working on this and more video tutorials. They'll be published at the MONAI YouTube channel.

Stay tuned :)
